contracts/src/teleporter/registry/TeleporterOwnerUpgradeable.sol

```diff
@@ -3,25 +3,31 @@
 
 // SPDX-License-Identifier: Ecosystem
 
-pragma solidity 0.8.18;
+pragma solidity 0.8.23;
 
 import {TeleporterUpgradeable} from "./TeleporterUpgradeable.sol";
-import {Ownable} from "@openzeppelin/contracts@4.8.1/access/Ownable.sol";
+import {OwnableUpgradeable} from
+    "@openzeppelin/contracts-upgradeable@5.0.2/access/OwnableUpgradeable.sol";
 
 /**
  * @dev Contract that inherits {TeleporterUpgradeable} and allows
  * only owners of the contract to update the minimum Teleporter version.
  *
  * @custom:security-contact https://github.com/ava-labs/teleporter/blob/main/SECURITY.md
  */
-abstract contract TeleporterOwnerUpgradeable is TeleporterUpgradeable, Ownable {
-    constructor(
+abstract contract TeleporterOwnerUpgradeable is TeleporterUpgradeable, OwnableUpgradeable {
+    // solhint-disable-next-line func-name-mixedcase
+    function __TeleporterOwnerUpgradeable_init(
         address teleporterRegistryAddress,
         address initialOwner
-    ) TeleporterUpgradeable(teleporterRegistryAddress) {
-        transferOwnership(initialOwner);
+    ) internal onlyInitializing {
+        __TeleporterUpgradeable_init(teleporterRegistryAddress);
+        __Ownable_init_unchained(initialOwner);
```

Comment:
you're right, I was originally thinking to use `unchained` to avoid cases where `Ownable_init` gets some added init's in its inheritance graph and to avoid collisions for new versions, but think the standard is to stick with `init` unless necessary to avoid collisions with `unchained`.

Reply:
updated